test/services/PageBuilderSpec.scala

```diff
@@ -122,6 +122,34 @@ class PageBuilderSpec extends BaseSpec with ProcessJson with StanzaHelper {
       }
     }
 
+    "detect SharedDataInputStanza error" in {
+      val flow = Map(
+        Process.StartStanzaId -> PageStanza("/url", Seq("1"), true),
+        "1" -> InstructionStanza(0, Seq("2"), None, false),
+        "2" -> InputStanza(Currency, Seq("4"), 0, Some(0), "Label", None, false),
+        "4" -> Choice(ChoiceStanza(Seq("5","end"), Seq(ChoiceStanzaTest("[label:label]", LessThanOrEquals, "8")), false)),
+        "5" -> PageStanza("/url2", Seq("1"), true),
+        "6" -> InstructionStanza(0, Seq("2"), None, false),
+        "2" -> InputStanza(Currency, Seq("4"), 0, Some(0), "Label", None, false),
```

Comment:
Can this flow be generated in Ocelot?  Duplicate id of "2"

Reply:
See viewer comment